content/docs/user-guide/integrations/sagemaker.md

```diff
@@ -125,6 +121,50 @@ The end result of running the pipeline looks like this:
 
 ![Pipeline](/img/sagemaker-pipeline.png)
 
+### Live experiment updates in SageMaker jobs
```

Comment:
It would be great to show / mention that is the simplest but a limited approach. Another alternative is to bundle git repo to Sagemaker, bundle results at the end of the job. I know users who are using that approach and it's not terribly complicated. We even have script examples. So we can potentially do a blog or cleanup and share them.

I'm not sure about Databricks though ...

Reply:
It would be great if we can see an example and we can add it. I hesitate to mention other approaches now if we don't have clear instructions on how to implement them yet.